app/models/data_resources/attraction.rb

```diff
@@ -17,7 +17,7 @@ class Attraction < ApplicationRecord
   has_one :contact, as: :contactable, dependent: :destroy
   has_many :media_contents, as: :mediaable, dependent: :destroy
   has_one :accessibility_information, as: :accessable, dependent: :destroy
-  has_one :operating_company, as: :companyable, dependent: :destroy
+  has_one :operating_company, as: :companyable, class_name: "OperatingCompany", dependent: :destroy
```

Comment:
weshalb ist der `class_name` jetzt notwendig? `"OperatingCompany"` ist doch genau camelcase von `: operating_company`?

Reply:
stimmt, ist vermutlich nur im genericItem notwendig.

Reply:
entfernt in https://github.com/ikuseiGmbH/smart-village-app-mainserver/pull/190/commits/ba1eec21473cea8c6cb0df13ac8641b224491d7a